tap-agent/Cargo.toml

```diff
@@ -37,10 +37,10 @@ ruint = { version = "1.12.3", features = [
 ], default-features = false }
 futures-util = { version = "0.3.28", default-features = false }
 jsonrpsee = { version = "0.24.0", features = ["http-client", "tracing"] }
-tap_aggregator = { git = "https://github.com/semiotic-ai/timeline-aggregation-protocol", rev = "eb8447e" }
 ractor = { version = "0.13", features = [
   "async-trait",
 ], default-features = false }
+tap_aggregator.workspace = true
 futures = { version = "0.3.30", default-features = false }
 
 [dev-dependencies]
```

Comment:
Could you add this change in a separate `chore` commit?

Reply:
Not really because it is an atomic change, this is require for all the crates to compile with the same versions

Reply:
you could have one `chore: update dependencies`, that is atomic and then the `feat: add dips support`.

Bumping tap_aggregator is not a feature to enable dips in tap-agent